### PR TITLE
Add gears of fortune.

### PR DESCRIPTION
**Pull request by Nahikariii**
It will look crap right now, till my latest pr is merged (because I've just updated the server :wink: )

### CODE REVIEW

File: gears-of-fortune.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@@include('./partials/header.html')
+
+<div class="section section-with-top">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-12">
+        <h1>Gears of fortune</h1>
+
+        <!-- Start Gamevy Embed code -->
+        <div class="game-embed game-embed--the-heist"
```
Comment (marekventur) [inline gears-of-fortune.html]: oops! (I don't think the class is used)

File: gears-of-fortune.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+             data-gamevy-embed
+             data-operator="lottoland"
```
Comment (grahamrhay) [inline gears-of-fortune.html]: shouldn't this match the one on the button?

Comment (marekventur) [inline gears-of-fortune.html]: That might have been my fault, the other ones probably need updating as well. What should we use for the homepage? gamevy?

Comment (grahamrhay) [inline gears-of-fortune.html]: yes

Comment (marekventur) [inline gears-of-fortune.html]: :ok:

**Pull request merged**